code/_helpers/mobs.dm

```diff
@@ -315,3 +315,11 @@ Proc for attack log creation, because really why not
 	else
 		. = getCompoundIcon(desired)
 		cached_character_icons[cachekey] = .
+
+//VOREStation Add Start

```

Comment:
We no longer require the vorestation add/remove comments. This was stopped being a requirement since summer. Details here: https://discord.com/channels/173831852602294272/828837425954160650/1123577273874911332

Reply:
Thanks, stumbled over it in many of the older files, will keep that in mind.